src/h5web/providers/silx/utils.ts

```diff
@@ -34,14 +34,11 @@ function buildTreeNode(
   };
 }
 
-export function buildTree(
-  metadata: SilxMetadata,
-  domain: string
-): TreeNode<HDF5Link> {
-  const rootLink: HDF5HardLink = {
-    class: HDF5LinkClass.Hard,
+export function buildTree(metadata: SilxMetadata): TreeNode<HDF5Link> {
+  const rootLink: HDF5RootLink = {
+    class: HDF5LinkClass.Root,
     collection: HDF5Collection.Groups,
-    title: domain,
+    title: '',
```

Comment:
I understand that the root is a special case but I thought that having the domain as title was making sense.

Can we have `'/'` for the root node instead of the empty string ?

Reply:
As discussed, the title isn't used at the moment and putting something else than an empty string may make it more difficult to build a path for each node during the construction of the tree.